mcs/class/System/winaot_System.dll.sources

```diff
@@ -1 +1 @@
-#include common.sources
+#include mobile_System.dll.sources
```

Comment:
The normal FileSystemWatcher.cs pulls in FAMWatcher.cs, KeventWatcher.cs and InotifyWatcher.cs. I remember there was some issue that DllImports can't be unresolved or something.

Anyway, this is something for the Windows team to look at, I just put in the mobile one for now as it was missing completely before and the typeforward from the facade was unresolved.